src/pages/generate/hooks/useOnnxModel.ts

```
@@ -218,14 +218,31 @@ export const preloadONNXModel = async (
  * - 640×640 렌더링 텐서를 입력으로 받아 감지 결과를 반환
  * - 추론 결과는 후속 파이프라인(`useFurnitureHotspots`)에서 원본 좌표로 보정
  */
-export function useONNXModel(modelPath: string) {
+interface UseONNXModelOptions {
+  enabled?: boolean;
+}
+
+export function useONNXModel(
+  modelPath: string,
+  options?: UseONNXModelOptions
+) {
```

ESLint

[error] Replace `⏎··modelPath:·string,⏎··options?:·UseONNXModelOptions⏎` with `modelPath:·string,·options?:·UseONNXModelOptions`

(prettier/prettier)